components/edit_post_modal/edit_post_modal.test.tsx

```diff
@@ -3,78 +3,95 @@
 
 import React from 'react';
 import ReactRouterEnzymeContext from 'react-router-enzyme-context';
+import {ShallowWrapper} from 'enzyme';
+import {IntlShape} from 'react-intl';
 
 import {isMobile} from 'utils/user_agent';
 
 import {shallowWithIntl} from 'tests/helpers/intl-test-helper';
 import {testComponentForLineBreak} from 'tests/helpers/line_break_helpers';
 import {Constants, ModalIdentifiers} from 'utils/constants';
 import DeletePostModal from 'components/delete_post_modal';
-import EditPostModal from 'components/edit_post_modal/edit_post_modal.jsx';
-import {testComponentForMarkdownHotkeys, makeSelectionEvent} from 'tests/helpers/markdown_hotkey_helpers.js';
+import EditPostModal, {Props} from 'components/edit_post_modal/edit_post_modal';
+import {
+    testComponentForMarkdownHotkeys,
+    makeSelectionEvent,
+} from 'tests/helpers/markdown_hotkey_helpers.js';
 import Textbox from 'components/textbox';
 
 jest.mock('actions/global_actions', () => ({
     emitClearSuggestions: jest.fn(),
 }));
 
 jest.mock('utils/user_agent', () => ({
-    ...jest.requireActual('utils/user_agent'),
+    ...jest.requireActual('utils/user_agent') as any,
```

Comment:
I am not sure how this should be typed. Any suggestions?

Reply:
I'm actually not sure we even need this field. If we do, it's okay to use `any` if we can't find a way to type it in tests. We try to avoid it, but it's okay in test files.

Reply:
I tried to remove this but the tests fail with the below error :slightly_frowning_face: 
```
TypeError: UserAgent.isSafari is not a function

      107 |         isKeyPressed(e, Constants.KeyCodes.ENTER) &&
      108 |         !e.shiftKey && // shift + enter is already handled everywhere, so don't handle again
    > 109 |         (e.altKey && !UserAgent.isSafari() && !cmdOrCtrlPressed(e)), // alt/option + enter is already handled in Safari, so don't handle again
          |                                 ^
      110 |     );
      111 | }
      112 |
```